general/marketing.rst

```diff
@@ -93,3 +93,21 @@ We are working on acquiring a `8', all over print, 3 sided custom tablecloth
 Check back for furthur details.
 
 
+Lessons Learned
```

Comment:
I wouldn't necessarily call this 'Lessons Learned' I would go moreso for S&T Branding rules. Talking about what you have here with a link to brand.mst.edu